platform/glue.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+
+# https://vaneyckt.io/posts/safer_bash_scripts_with_set_euxo_pipefail/
+set -euxo pipefail
+
+echo "Removing old glue files..."
+rm -rf platform/glue
+
+echo "Generating glue files..."
+roc glue ../roc/crates/glue/src/RustGlue.roc platform/glue platform/main-glue.roc
+
+echo "NOTE: manually fix any errors in platform/glue files"
```

Comment:
There are issues such as [#6012](https://github.com/roc-lang/roc/issues/6012) which can be manually fixed when re-generating glue.